docs/usage.rst

```diff
@@ -1265,6 +1265,119 @@ Besides the `invest` variable, new variables are introduced as well. These are:
       monthly periods, but you would need to be very careful in parameterizing your energy system and your model and also,
       this would mean monthly discounting (if applicable) as well as specifying your plants lifetimes in months.
 
+Modelling cellular energy systems
```

Comment:
You could make it more general by adding " and modularizing energy system models".